js/components/TournamentList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {requestTournamentList} from '../api';
+
+export default class TournamentList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
```

Comment:
It cloud be used to show the user an error originating from the backend. Displaying this error is currently not implemented. Since I don't know what kind of error (that is meaningful to the user) could occur there, I will remove the currently unused error code.